docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -32,7 +32,7 @@ When a FILTER clause is attached to an aggregate function, only the matching row
 GROUP BY group_expression [ , group_expression [ , ... ] ]
     [ { WITH ROLLUP | WITH CUBE | GROUPING SETS (grouping_set [ , ...]) } ]
 
-GROUP BY GROUPING SETS (grouping_set [ , ...])
+GROUP BY group_expression_with_grouping_analytics [ , group_expression_with_grouping_analytics [ , ... ] ]
```

Comment:
`group_expression_with_grouping_analytics` is hard to read, let's inline it
```
GROUP BY { group_expression | { ROLLUP | CUBE | GROUPING SETS } (grouping_set [ , ...]) } [ , ... ]
```

Reply:
Done